### PR TITLE
Feature Weather App

### PR DESCRIPTION
**Pull request by sergiocortessat**
Pull request 1 Weather app.

Hello TSE. Thanks for checking my project

As follow, here is the Weather App build with Javascript, Webpack, and Bootstrap. As such the app complies with the minimum requirements such as weather forecast using the weather API. Search for a specific location and toggle displaying the data in Fahrenheit or Celsius.

- Page can take any city name
- User will be displayed with the weather, city name, current conditions, temperature, and real thermal feeling
- The background on the page changes dynamically to fit the condition weather.
- The user can display the page in Celcius or Fahrenheit

### CODE REVIEW

File: src/dom.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      innerRealTempElement.textContent = parsedWeather.fCondition;
+      innerRealTempElement.innerHTML += '&#8457';
+      realTempDiv.append(innerRealTempName, innerRealTempElement);
+      tempButton.innerText = 'CELSIUS';
+    } else {
+      tempButton.classList.add('cel');
+      tempDiv.innerHTML = '';
+      realTempDiv.innerHTML = '';
+      tempDiv.append(tempName, tempElement);
+      realTempDiv.append(realTempName, realTempElement);
```
Comment (SajjadAhmad14) [inline src/dom.js]: App logic should not be mixed with DOM manipulation.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
# Changes Required :recycle:
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hello, you have been doing a tremendous job so far :ok_hand:
## To Highlight 🎉
- [x] You have mastered Git flow.
- [x] PR title and summary are descriptive.
- [x] App UI is pleasing.
## To Improve 🙏
- [x] Please add the `npm install` command in your `Clone project` section of README. The project can not be installed and run without this command.
- [x] LinkedIn link is not active in the `Authors` section. Please fix that.
- [x] App logic should be separate from DOM manipulation tasks.
- [x] async function should handle error so that it does not crash if an error occurs. Your app should show a friendly error when the user inputs an invalid city name.
- [x] Please see inline comments for reference.
# Optional
- [x] It's nice if the app fetches the current geolocation of the user when the it is first accessed and fetches the weather from that location
### Happy Coding ☺️

## If you have any questions you can comment here in this PR or can always connect with me on slack ( @Sajjad Ahmad ).

**Comment by sergiocortessat** [comment]
Pull request 2

Hello TSE and thank you for checking on my project. I have implemented the requested changes to the repo such as:

- npm install command in your Clone project section of README.
- LinkedIn link has been fixed
- App logic was separated from DOM.
- The async function nows handle a pop up error

**Review by NtwaliHeritier — APPROVED** [review]
### **STATUS: APPROVED✔✔✔✔✔**

Hello @sergiocortessat 

Great work making the requested changes🙌👏👏
You can merge👌👍

Happy coding💻💻

**Pull request merged**